test/e2e/util/env/env.go

```diff
@@ -51,3 +52,7 @@ func GetNrAccountId() int {
 func GetNrApiBaseUrl() string {
 	return getEnvVar(NrApiBaseUrl)
 }
+
+func IsFipsMode() bool {
+	return getEnvVar(FipsMode)
```

Comment:
This currently does not compile. The easiest is probably to do something like
```suggestion
	return getEnvVar(FipsMode) == "true"
```
One could also leverage `strconv.ParseBool` but that includes dealing with the error handling if the value is not from a well-defined list. We can probably save ourselves that complex in this case.